src/resources/postcss/views/_list.pcss

```diff
@@ -49,6 +51,11 @@
 		}
 	}
 
+	.tribe-events-calendar-list__event--title-link {
```

Comment:
So we need to follow BEM styles with this. It should probably be something like `.tribe-events-calendar-list__event-title-link`. The `--` is reserved for the M (modifier) of BEM, where if we have different variations of the same element. If we had a red link in addition to the regular link, for example, we might have a class `.tribe-events-calendar-list__event-title-link--red`.